dask_kubernetes/operator/tests/resources/simplecluster.yaml

```diff
@@ -16,7 +16,8 @@ spec:
     # affinity: null
     # tolerations: null
     # serviceAccountName: null
-  workers:
-    replicas: 3
-    resources: {}
-    env: {}
\ No newline at end of file
+  replicas: 3
+  resources: {}
+  env: {}
+status:
+  replicas: 3
```

Comment:
I don't think we should be setting any `status` config here, this should be generated by Kubernetes.